api/src/pcapi/core/offerers/api.py

```diff
@@ -3147,3 +3147,23 @@ def synchronize_from_ds_and_check_application(offerer_id: int) -> bool:
         .filter(offerers_models.Venue.collectiveDmsApplications.any())
     )
     return db.session.query(query.exists()).scalar()
+
+
+def create_action_history_when_move_offers(
+    origin_venue_id: int,
+    destination_venue_id: int,
+    offer_ids: list[int] | None,
+    offers_type: typing.Literal["individual", "collective"],
+) -> None:
+    """
+    Simple action history entry, in both origin and destination venues, when moving
+    all offers (individual or collective) from one venue to another.
+    """
+    extra_data = {offers_type + "_offer_ids": offer_ids}
+    action_history_origin = history_models.ActionHistory(
+        venueId=origin_venue_id, actionType=history_models.ActionType.MOVE_ALL_OFFER, extraData=extra_data
```

Comment:
est-ce qu'on veut pas l'inverse justement ? 
```suggestion
        venueId=destination_venue_id, actionType=history_models.ActionType.MOVE_ALL_OFFER, extraData=extra_data
```

Reply:
Je me remds compte qu'il manque peut être l'info de où sont allées (et d'où viennent) les offres. Par contre le venueId est bon pour moi.

Reply:
ah oui, ok du coup il faut juste ajouter un source / destination dans les extra data